hoppfish/hwangsae-recorder-user-command.py

```diff
@@ -63,7 +63,89 @@ def post(self, method, edge_id):
             http_ret = 404
         return response, http_ret
 
+class vod(Resource):
+    def get(self, method, edge_id = '', file_id = ''):
+        http_ret = 201
+        try:
+            obj = dbus.SessionBus().get_object('org.hwangsaeul.Hwangsae1.RecorderAgent',
+                    '/org/hwangsaeul/Hwangsae1/RecorderInterface')
+            dbus_interface = dbus.Interface(obj,
+                    dbus_interface='org.hwangsaeul.Hwangsae1.RecorderInterface')
+
+            ret = ''
+            if method == 'lookup-by-record':
+                record_id = request.args.get('recordId')
+                try:
+                    range_from = int(request.args.get('from'))
+                except:
+                    range_from = 0
+                try:
+                    range_to = int(request.args.get('to'))
+                except:
+                    range_to = 0
+                ret = dbus_interface.LookupByRecord(record_id, range_from, range_to)
+                edge_id = ret[0]
+                file_list = ret[1]
+                ret = {}
+                ret['edgeId'] = edge_id
+                file_list_ret = []
+                for f in file_list:
+                    file_data = {}
+                    file_data['fileId'] = f[0]
+                    file_data['start'] = f[1]
+                    file_data['end'] = f[2]
+                    file_data['size'] = f[3]
```

Comment:
Just a matter of preference, but I always found this notation more elegant
```
file_data = {
  'fileId': f[0],
  'start': f[1],
  'end': f[2],
  'size': f[3]
}
```
Less typing and less repetitive, and the way the code is formatted is visually pretty close to the JSON output.

Or you can go even fancier with Python list comprehensions instead of a standard for loop :)

```
ret['fileList'] = [
  {
    'fileId': f[0],
    'start': f[1],
    'end': f[2],
    'size': f[3]
  } for f in file_list
]
```

Reply:
Sure, let do it fancier! I like you first suggestion.